docs/routing.md

```diff
@@ -24,6 +24,13 @@ DNS resolution of host names is external to the OpenShift system.
 Please note, that starting the router in the vagrant environment requires it to be pulled into docker.  This may take some time.
 Once it is pulled it will start and be visible in the `docker ps` list of containers and your pod will be marked as running.
 
+A router requires a service account that has access to a security context constraint which allows host ports.
```

Comment:
Trying to piece this together, I'm guessing I need an SCC with something the `cap` column documented here: https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html.  Can you provide that value so that I can know which SCC I should use or create?